cmd/util/cmd/execution-state-extract/execution_state_extract.go

```diff
@@ -33,7 +33,7 @@ func extractExecutionState(dir string, targetHash flow.StateCommitment, outputDi
 
 	newState, err := led.ExportCheckpointAt(targetHash,
 		[]ledger.Migration{},
-		[]ledger.Reporter{migrations.StorageReporter{Log: log, OutputDir: outputDir}},
+		[]ledger.Reporter{migrations.ContractReporter{Log: log, OutputDir: outputDir}, migrations.StorageReporter{Log: log, OutputDir: outputDir}},
```

Comment:
Minor: Maybe but the items/reporters on separate lines to make this more readable